mappings/net/minecraft/world/gen/chunk/ChunkGenerator.mapping

```diff
@@ -1,4 +1,6 @@
 CLASS net/minecraft/class_2794 net/minecraft/world/gen/chunk/ChunkGenerator
+	COMMENT In charge of shaping, adding biome specific surface blocks, carving, and decorating chunks,
+	COMMENT as well as populating {@link net.minecraft.world.biome.Biome biomes} and {@link net.minecraft.entity.Entity entities}.
```

Comment:
```suggestion
	COMMENT as well as populating {@linkplain net.minecraft.world.biome.Biome biomes} and {@linkplain net.minecraft.entity.Entity entities}.
```
You should use non-monospace links for plain text.

Reply:
irritatered